client_cli/pytests/src/client_cli/iroha.py

```diff
@@ -48,103 +43,57 @@ def should(self, _expected):
         """
         return self
 
-    def domains(self):
+    def domains(self) -> List[str]:
         """
-        Retrieve domains from the Iroha network and store them in the _domains attribute.
+        Retrieve domains from the Iroha network and return then as list of ids.
 
         :return: The current Iroha object.
         :rtype: Iroha
```

Comment:
Now it doesn't conform to the new signature. This is applicable to all functions refactored below in the same file.